deployments/mainnet/configuration.json

```diff
@@ -0,0 +1,74 @@
+{
+  "symbol": "📈USDC",
```

Comment:
Just a heads-up: if #498 gets merged before this PR, we'll need to move these files into `deployments/mainnet/usdc/`